packages/web-components/src/components/cbp-structured-list/cbp-structured-list.stories.tsx

```diff
@@ -409,28 +409,206 @@ StructuredListMediaCalendar.argTypes = {
     control: 'object',
   },
 }
-StructuredListMediaCalendar.args = { //todo: update text in samples to match calendar text
+StructuredListMediaCalendar.args = {
   showHeader: false,
   listItems: [
     {
-      content: "<cbp-grid grid-template-columns='1fr 5fr' gap='1rem'><div class='cbp-structured-list-calendar-item'><span class='cbp-structured-list-calendar-item-month'>apr</span><span class='cbp-structured-list-calendar-item-day'>31</span></div><div><cbp-typography tag='p'><cbp-icon name='arrow-right' color='var(--cbp-link-color)' size='1rem'></cbp-icon><cbp-link href='#' target='_self' style='margin-inline-start: 0.5rem'>Find the Hidden Beetle Training</cbp-link></cbp-typography><cbp-typography tag='p'>This training session will help you find those pesky hiding beetles. Inside boxes, on top of containers, or in someone's pocket, these bugs don't stand a chance of getting to America's sweet, delicious forests</cbp-typography><cbp-typography tag='p'><cbp-icon name='globe' size='1rem'></cbp-icon> <i>https://www.cbp.dhs.gov/news/beetles</i></cbp-typography></div><cbp-grid>",
+      content: `<cbp-grid grid-template-columns='1fr 5fr' gap='1rem'>
```

Comment:
This grid should still not have a 1fr for the first column. It should be fixed, but in relative units (rems). If you view this on a large screen, the gap between the calendar and text grows, which is not ideal.

Reply:
updated to 6.25rem 1fr